### PR TITLE
[14.0][FIX] stock_barcodes: make actions readable by non-admin users.

### PR DESCRIPTION
**Pull request by LoisRForgeFlow**
leftover from migration.

@ForgeFlow

cc @sergio-teruel

### CODE REVIEW

**Review by pedrobaeza — APPROVED** [review]

**Comment by LoisRForgeFlow** [comment]
@pedrobaeza I added one more change, please check the latest commit and update your review if needed.

File: stock_barcodes/wizard/stock_barcodes_read_picking.py [hunk 1]
```diff
@@ -254,7 +254,7 @@ def _prepare_move_line_values(self, candidate_move, available_qty):
             "picking_id": picking.id,
             "move_id": candidate_move.id,
             "qty_done": available_qty,
-            "product_uom_id": self.product_id.uom_po_id.id
+            "product_uom_id": candidate_move.product_uom.id or self.product_id.uom_id.id
```
Comment (pedrobaeza) [inline stock_barcodes/wizard/stock_barcodes_read_picking.py:257]: Shouldn't this depend on the type of picking? cc @sergio-teruel

Comment (LoisRForgeFlow) [reply stock_barcodes/wizard/stock_barcodes_read_picking.py:257]: Even if it is a receipt it does not imply it is a purchase. Using the PO UoM looks like the less frequent case here. If we have a move and the move comes from a PO, the move will already have the PO UoM as `product_uom`. Covering more complex cases would need a dependency with purchase I think.

Comment (pedrobaeza) [reply stock_barcodes/wizard/stock_barcodes_read_picking.py:257]: OK, I think it's correct to use the move UoM, but then the `or` is never going to be used, as it always has a value. Let's see Sergio to tell us why he put such UoM.

**Review by sergio-teruel — APPROVED** [review]

File: stock_barcodes/wizard/stock_barcodes_read_picking.py (hunk 1 above)
Comment (LoisRForgeFlow) [reply stock_barcodes/wizard/stock_barcodes_read_picking.py:257]: There is actually a case in which is going to be used: when you create a new non-planned picking, in that case you start to scan operations and there is no stock move yet until you validate the operation you just scanned.

Comment (pedrobaeza) [reply stock_barcodes/wizard/stock_barcodes_read_picking.py:257]: OK, maybe in that case the UoM should depend on the picking type code.

Comment (pedrobaeza) [reply stock_barcodes/wizard/stock_barcodes_read_picking.py:257]: Uhm, thinking twice, no, it should always put the stock UoM, not the PO one, so it's correct.

**Pull request merged**